src/views/Game/PlayerCards.tsx

```diff
@@ -301,7 +301,7 @@ function PlayerCard({
                             </div>
                         )}
                         <div className="player-flag">
-                            <Flag country={player.country ?? "un"} />
+                            <Flag country={player.country} />
```

Comment:
Looks like if we want to get this working for non-rengo, we can just pull `country` off historical:

```
<Flag country={player.country || historical?.country} />
```

So just one more option..